### PR TITLE
Adding Week 11 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding Weekly post

image

### CODE REVIEW

File: docs/source/posts/2022/2022-08-17-week-11-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+.. post:: August 17 2022
+   :author: Praneeth Shetty 
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+This week I tried to imitate the working of `vtkImageTracer`. Previously I had created a small prototype for freehand drawing by adding points at the mouse position (which you can check out `here <https://blogs.python-gsoc.org/en/ganimtron_10s-blog/week-7-working-on-rotation-pr-and-trying-freehand-drawing/>`_). As mentioned there were some drawback of this method.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-17-week-11-praneeth.rst:13]: Missing a comma after `As mentioned`

File: docs/source/posts/2022/2022-08-17-week-11-praneeth.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+So to overcome these I tried combining both methods. Considering points using the previous method and instead of adding points I tried creating lines between them which looks promising. Below you can see a demo.
+
+`Freehand Drawing: <https://github.com/ganimtron-10/fury/tree/freehand-drawing>`_
+
+.. image:: https://user-images.githubusercontent.com/64432063/186952329-636a0d81-6631-4e8d-9486-9a8c5e88a9a7.gif
+    :width: 400
+    :align: center
+
+While doing this I started working on how I could efficiently draw the lines and smoothen them out. My mentors referred me `this <https://github.com/rougier/python-opengl/blob/master/09-lines.rst>`_  to learn more about constructing and rendering lines.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-17-week-11-praneeth.rst:22]: Missing a comma after `While doing this`

**Review by m-agour — COMMENTED** [review]
Thanks @ganimtron-10 for this nice blog.
Please consider my review below.

File: docs/source/posts/2022/2022-08-17-week-11-praneeth.rst (hunk 1 above)
Comment (xtanion) [reply docs/source/posts/2022/2022-08-17-week-11-praneeth.rst:13]: ++

File: docs/source/posts/2022/2022-08-17-week-11-praneeth.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+.. post:: August 17 2022
+   :author: Praneeth Shetty 
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+This week I tried to imitate the working of `vtkImageTracer`. Previously I had created a small prototype for freehand drawing by adding points at the mouse position (which you can check out `here <https://blogs.python-gsoc.org/en/ganimtron_10s-blog/week-7-working-on-rotation-pr-and-trying-freehand-drawing/>`_). As mentioned there were some drawback of this method.
+So to overcome these I tried combining both methods. Considering points using the previous method and instead of adding points I tried creating lines between them which looks promising. Below you can see a demo.
```
Comment (xtanion) [inline docs/source/posts/2022/2022-08-17-week-11-praneeth.rst:14]: Missing comma after `these`

**Review by xtanion — COMMENTED** [review]
HI @ganimtron-10, The blog LGTM. Just few commas are missing, PTAL.
Thanks

**Comment by ganimtron-10** [comment]
Thank you for the review @m-agour and @xtanion !!

**Comment by skoudoro** [comment]
Thank @ganimtron-10, LGTM, merging

**Pull request merged**